test/front_matter.jl

```diff
@@ -112,6 +113,9 @@ using .TestUtils:
     test_tangent,
     check_allocs
 
+using Test: TestLogger
+using Logging: ConsoleLogger
+
```

Comment:
Actually, these `using`s are redundant, because we `using` both `Test` and `Logging` above, and the names are exported.

Reply:
```suggestion
```